resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,428 @@
+openapi: 3.0.3
+info:
+  title: CTC Traders Test Support
+  contact: {}
+  version: '2.0'
+  description: |
+    # Overview
+    This API allows testers to inject departure and arrival movement notifications as 
+    if they have come from the office of departure or the office of destination by the New Computerised Transit System (NCTS).

```

Comment:
Use the CTC Traders Test Support API to inject departure and arrival movement notifications as if they have come from a customs office of departure or destination through the New Computerised Transit System (NCTS).